.github/workflows/clean_install_examples.yml

```
@@ -0,0 +1,28 @@
+# This workflow checks that examples can be installed cleanly.
+# It only runs on PRs that affect the `examples/` directory.
+
+name: Check examples install status
+
+on: 
+  pull_request:
+    paths:
+      - 'examples/**'
+
+jobs:
+  install:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+
+      - name: Check install status
+        run: |
+          echo "If this step fails, reproduce locally by running `npm run clean:node && cd examples && npx lerna clean --yes` first"
```

```suggestion
          echo "If this step fails, reproduce locally by running `npm run clean:node && cd examples && npx lerna clean --yes && npm ci`"
```

I think it makes sense to just give the whole command if you're already giving most of it!

---

you're right. i changed this. also, the backticks were causing the commands to actually get executed lol so i fixed that, and moved the message to a step after the ci step. because otherwise that message is all the way at the beginning and easy to miss if you don't scroll up looking for it.